.github/workflows/test.yml

```diff
@@ -30,34 +32,49 @@ jobs:
           # Use .ci/.matrix_framework_full.yml if it's a scheduled workflow, otherwise use .ci/.matrix_framework.yml
           frameworksFile: .ci/.matrix_framework${{ github.event_name == 'schedule' && '_full' || '' }}.yml
           excludedFile: .ci/.matrix_exclude.yml
+      - name: Split matrix
+        shell: python
+        id: split
+        run: |
+          import os
+          import json
+
+          def split(lst, n):
+            return [lst[i::n] for i in range(n)]
+
+          matrix = json.loads(os.environ['GENERATED_MATRIX'])
+          chunks = split(matrix['include'], 4)
```

Comment:
What do you think adding some comments regarding this value and how those fixed jobs use it? Just we remember in the future how it works

Reply:
https://github.com/elastic/apm-agent-python/pull/1747/commits/d5bac7fbd49f1f0f70b3c4a28becad628b2e70eb